common/traits/000_wc_leadership_traits.txt

```diff
@@ -0,0 +1,195 @@
+# Written by: DukeJeans
+# Created: 1/22/2019
+# Modified: 3/28/2019
+#Leadership traits
+
+#Warcraft
+archmagus_leader = {
+	leader = yes
+	random = no
+	customizer = no
+
+	command_modifier = {
+		light_infantry = 0.2
+	}
+}
+
+light_foot_leader = {
+	leader = yes
+
+	command_modifier = {
+		heavy_infantry = 0.2
```

Comment:
Use actual_heavy_infantry, heavy_infantry gives bonus to pikeman and heavy infantry